content/docs/user-guide/external-dependencies.md

```diff
@@ -88,7 +91,7 @@ $ dvc run -n download_file \
 $ dvc run -n download_file \
           -d ssh://user@example.com/path/to/data.txt \
           -o data.txt \
-          scp user@example.com:/path/to/data.txt data.txt
+          scp ssh://user@example.com:/path/to/data.txt data.txt
```

Comment:
hmm, I don't usually specify the protocol like this

Reply:
Right. Reverting this change.